www/pfSense-pkg-vHosts/README.md

```diff
@@ -0,0 +1,58 @@
+# vHosts
```

Comment:
Ok, this one has me baffled. README.md is not included in the build and only serves as a help page for someone that might want to have a look at some documentation. The help page facility for packages is broken which is why there is no help for many packages. In the future, I'm hoping to fix that and use this file to create the actual help page. In the meantime, what is the objection to putting this file in the root given it's not included in the build?